jwst/ami/lg_model.py

```diff
@@ -496,7 +460,7 @@ def improve_scaling(
         self.corrs = np.zeros(len(radlist))
 
         self.rots = radlist
-        for q, rad in enumerate(radlist):
+        for q, _rad in enumerate(radlist):
```

Comment:
The variable `_rad` is never used, so this could just be `for q in range(len(radlist)):`.

Reply:
good point will fix